### PR TITLE
Added css filters to frontend/ui-generators

### PR DESCRIPTION
**Pull request by Swapnil-Singh-99**
## Fixes Issue
Closes : #1110 

## Changes proposed

I have added CSS Filters to Frontend/UI Generators

## Screenshots

image

## Note to reviewers
Review and Merge this PR under GSSOC'23

### CODE REVIEW

**Comment by rupali-codes** [comment]
@Swapnil-Singh-99 please add _closes #1110_ in the description of this PR and also resolve the merge conflicts :)

**Comment by Swapnil-Singh-99** [comment]
@rupali-codes I have resolved conflicts

File: database/frontend/ui-generators.json
Comment (CBID2) [inline database/frontend/ui-generators.json:172]: [Proofreading]
```suggestion
    "description": "This is a visual playground for generating CSS for custom and Instagram-like photo filters. Experiment with your own uploaded photo or select one from the Unsplash collection.",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @Swapnil-Singh-99. I just found some proofreading errors in your description. Other than that, you are off to a great start! :)

File: database/frontend/ui-generators.json
Comment (Swapnil-Singh-99) [reply database/frontend/ui-generators.json]: @CBID2 I have Resolved it
Please review

**Review by CBID2 — DISMISSED** [review]
LGTM

**Comment by k-deepak04** [comment]
@Swapnil-Singh-99 please resolve the conflicts

**Review by k-deepak04 — CHANGES_REQUESTED** [review]
manage your changes with respect to the cureent updates,
try rebasing your code with the main branch and then add commits to prevent conflicts, rest all good.

**Comment by Swapnil-Singh-99** [comment]
> manage your changes with respect to the cureent updates, try rebasing your code with the main branch and then add commits to prevent conflicts, rest all good.

@k-deepak04 Done Please review

**Review by k-deepak04 — APPROVED** [review]

**Comment by k-deepak04** [comment]
@CBID2 please review and merge it

**Review by CBID2 — APPROVED** [review]
LGTM

**Comment by CBID2** [comment]
> @CBID2 please review and merge it

I can't merge it for some reason

**Comment by k-deepak04** [comment]
> > @CBID2 please review and merge it
> 
> I can't merge it for some reason

i guess until rupali reviews this we cannot merge it

so @rupali-codes please review this

**Review by rupali-codes — APPROVED** [review]

**Pull request merged**
